packages/components/src/components/CreateCard/CreateCard.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Icon, IconNames } from '../Icon';
+import { Stack } from '../Stack';
+import { Text } from '../Text';
+
+type StackProps = React.ComponentProps<typeof Stack>;
+
+type CreateCardProps = {
+  icon: IconNames;
+  label: string;
+} & StackProps;
+
+export const CreateCard: React.FC<CreateCardProps> = ({
+  as = 'button',
+  icon,
+  label,
+  ...props
+}) => {
+  return (
+    <Stack
+      as={as}
+      css={{
+        position: 'relative',
+        overflow: 'hidden',
+        maxWidth: '276px',
+        padding: '24px',
+        flex: 1,
+        justifyContent: 'space-between',
+        border: '1px solid',
+        borderColor: 'transparent',
+        borderRadius: '4px',
+        backgroundColor: '#161616',
+        outline: 'none',
+        textDecoration: 'none',
+        transition: 'background ease-in-out, opacity ease-in-out',
+        transitionDuration: theme => theme.speeds[2],
+        '&:hover, &:active': {
+          backgroundColor: 'card.backgroundHover',
+        },
+        '&:focus-visible': {
+          borderColor: 'focusBorder',
```

Comment:
Why `borderColor` instead of `outline`?

Reply:
@tristandubbeld the other cards are using the border because the styles are similar if the card is selected, so I choose to use the border for consistency.

Reply:
What do you mean exactly with selected? For the sandbox cards when they're selected through e.g. multi-select you mean? Because that makes sense. I was thinking about this a bit and I think that we should have different focus and selected styles for those cards.